tests/functional/test_home.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.home import HomePage
+
+
+@pytest.mark.nondestructive
+def test_promos_are_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.length == 16
+
+
+@pytest.mark.nondestructive
+def test_tweet_is_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.tweet_is_present
+
+
+@pytest.mark.nondestructive
+def test_download_button_is_displayed(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.download_button_is_displayed
```

Comment:
Nit: Sometimes we're using `is_x_displayed` and sometimes we're using `x_is_displayed`. Eventually these inconsistencies are bound to creep in, I just thought I'd flag it as I noticed it.